### PR TITLE
DCOS_OSS-2206: Add enzyme to 1.11

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
I just added enzyme to the 1.11 branch without porting all the tests over (massive merge conflicts there).

You can test it by doing `npm install && npm test`.

I flag it as a blocker as it blocks a blocker, I hope this is OK

**Checklist**
- [X] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,8 +92,9 @@
     "conventional-changelog-lint-config-angular": "0.4.1",
     "css-loader": "0.23.1",
     "cypress": "2.1.0",
+    "enzyme": "3.3.0",
+    "enzyme-adapter-react-15.4": "1.0.5",
```
Comment (Fabs) [inline package.json]: I have nuked my nodes folder, ran:
nvm install (switches to the right node)
npm install
npm test

image

All tests failed becauese of that.

Comment (Fabs) [reply package.json]: Forget about it, it was solved.

Comment (DanielMSchmidt) [reply package.json]: Yeah, ran into the same thing previously 😞

**Comment by Fabs** [comment]
I had the wrong plugins branch. Test pass for me now.

**Comment by DanielMSchmidt** [comment]
run integration tests

**Pull request merged**